org/aeryn.ts

```diff
@@ -0,0 +1,47 @@
+// For the inspiration for this file, see: https://github.com/danger/peril-settings/blob/fc0015452438c8a1624c0951a600f723f2e60fea/org/aeryn.ts#L18-L39
+import { schedule, danger, markdown } from "danger"
+
+// Hey there!
+//
+// When a PR is closed, this file gets run. The purpose of this file is to 
+// replicate Aeryn (https://github.com/Moya/Aeryn), which invites new 
+// contributors to join the organization after their first PR gets merged.
+//
+// Ignore the next four const lines.
+// The inspiration for this is https://github.com/artsy/artsy-danger/blob/f019ee1a3abffabad65014afabe07cb9a12274e7/org/all-prs.ts
+const isJest = typeof jest !== "undefined"
+// Returns the promise itself, for testing.
+const _test = (reason: string, promise: Promise<any>) => promise
+// Schedules the promise for execution via Danger.
+const _run = (reason: string, promise: Promise<any>) => schedule(promise)
+const wrap: any = isJest ? _test : _run
+
+export const aeryn = wrap("When a PR is merged, check if the author is in the org", async () => {
```

Comment:
This `wrap` function is for testing, which we don't have yet in this repo. I figured it'd be easier to leave it in than take it out, and now it can serve as an example for other files.

Reply:
Awesome! The goal is to dig into Jest/testing this weekend so this will help me a lot. Testing is pretty much blocking every issue.